lib/ui/text.dart

```diff
@@ -1682,6 +1744,67 @@ class ParagraphBuilder extends NativeFieldWrapperClass2 {
   }
   String _addText(String text) native 'ParagraphBuilder_addText';
 
+  /// Adds an inline placeholder space to the paragraph.
+  ///
+  /// The paragraph will contain a rectangular space with no text of the dimensions
+  /// specified.
+  ///
+  /// The `width` and `height` parameters specify the size of the placeholder rectangle.
+  ///
+  /// The [alignment] parameter specifies how the placeholder rectangle will be vertically
```

Comment:
alignment, baseline, baselineOffset (2x) and height should be enclosed in ` instead of [] since they are just arguments to the method and not properties.